<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPESectionCharacteristics</class>
 <widget class="QDialog" name="DialogPESectionCharacteristics">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>302</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>510</width>
    <height>302</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>510</width>
    <height>310</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Section Characteristics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBoxX" name="checkBoxReadOnly">
       <property name="text">
        <string>Read only</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEditX" name="lineEditSectionSectionCharacteristics">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBoxX" name="comboBoxALIGN">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBoxX" name="checkBoxCNT_CODE">
        <property name="text">
         <string notr="true">CNT_CODE</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBoxX" name="checkBoxMEM_DISCARDABLE">
        <property name="text">
         <string notr="true">MEM_DISCARDABLE</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBoxX" name="checkBoxCNT_INITIALIZED_DATA">
        <property name="text">
         <string notr="true">CNT_INITIALIZED_DATA</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBoxX" name="checkBoxMEM_NOT_CACHED">
        <property name="text">
         <string notr="true">MEM_NOT_CACHED</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBoxX" name="checkBoxCNT_UNINITIALIZED_DATA">
        <property name="text">
         <string notr="true">CNT_UNINITIALIZED_DATA</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBoxX" name="checkBoxMEM_NOT_PAGED">
        <property name="text">
         <string notr="true">MEM_NOT_PAGED</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBoxX" name="checkBoxLNK_INFO">
        <property name="text">
         <string notr="true">LNK_INFO</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBoxX" name="checkBoxMEM_SHARED">
        <property name="text">
         <string notr="true">MEM_SHARED</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBoxX" name="checkBoxLNK_REMOVE">
        <property name="text">
         <string notr="true">LNK_REMOVE</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBoxX" name="checkBoxMEM_LOCKED">
        <property name="text">
         <string notr="true">MEM_LOCKED</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBoxX" name="checkBoxMEM_EXECUTE">
        <property name="text">
         <string notr="true">MEM_EXECUTE</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBoxX" name="checkBoxLNK_COMDAT">
        <property name="text">
         <string notr="true">LNK_COMDAT</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBoxX" name="checkBoxMEM_PRELOAD">
        <property name="text">
         <string notr="true">MEM_PRELOAD</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QCheckBoxX" name="checkBoxMEM_READ">
        <property name="text">
         <string notr="true">MEM_READ</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBoxX" name="checkBoxNO_DEFER_SPEC_EXC">
        <property name="text">
         <string notr="true">NO_DEFER_SPEC_EXC</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBoxX" name="checkBoxLNK_NRELOC_OVFL">
        <property name="text">
         <string notr="true">LNK_NRELOC_OVFL</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QCheckBoxX" name="checkBoxMEM_WRITE">
        <property name="text">
         <string notr="true">MEM_WRITE</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBoxX" name="checkBoxGPREL">
        <property name="text">
         <string notr="true">GPREL</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBoxX" name="checkBoxMEM_PURGEABLE">
        <property name="text">
         <string notr="true">MEM_PURGEABLE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonOK">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonApply">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPushButtonX</class>
   <extends>QPushButton</extends>
   <header>qpushbuttonx.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEditX</class>
   <extends>QLineEdit</extends>
   <header>qlineeditx.h</header>
  </customwidget>
  <customwidget>
   <class>QComboBoxX</class>
   <extends>QComboBox</extends>
   <header>qcomboboxx.h</header>
  </customwidget>
  <customwidget>
   <class>QCheckBoxX</class>
   <extends>QCheckBox</extends>
   <header>qcheckboxx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
